specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/iotSensors.json

```diff
@@ -0,0 +1,308 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2020-08-06-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/iotSensors": {
```

Comment:
>Microsoft.Security/iotSensors [](start = 47, length = 29)

Can you explain why IOT objects are modeled under Microsoft namespace?

Reply:
IoT Defender is a new security service that specializes in enhancing security/discoverability of IoT devices.
In Service Tree, it's located under `Cloud + AI Platform \ Security \ Azure IoT Security`, and it's resources are available under the `Microsoft.Security` provider (that is, sharing the provider with Azure Security Center).